db/migrate/20231218155252_add_help_center_site_setting.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class AddHelpCenterSiteSetting < ActiveRecord::Migration[7.1]
+  def up
+    return if SiteSetting.exists?(setting: Setting.find_by(name: 'HelpCenter'))
+
+    SiteSetting.create!(
+      setting: Setting.find_by(name: 'HelpCenter'),
+      value: '',
+      provider: 'greenlight'
+    )
```

Comment:
For simplicity, you can merge this with the migration above

Reply:
Got it, made the change and pushed the combined migration file.